public/app/features/annotations/standardAnnotationSupport.ts

```diff
@@ -23,7 +23,7 @@ export const standardAnnotationSupport: AnnotationSupport = {
   /**
    * Assume the stored value is standard model.
    */
-  prepareAnnotation: (json: any) => {
+  prepareAnnotation: (json: AnnotationQuery) => {
```

Comment:
the upstream interface is:
https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/types/annotations.ts#L91

and the goal is literally to convert whatever the old totally untyped approach was into an explicit query.    In many cases this is just a string, but really had no constraints since the beginning of annotations.  In our code, this type will work fine -- but i fear it gives the impression that we know what the input is and are not explicitly converting it to an annotation

Reply:
ah gotcha 👌

I'll undo this one next week